src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/.template.config/ide.host.json

```diff
@@ -1,22 +1,7 @@
 {
   "$schema": "http://json.schemastore.org/vs-2017.3.host",
-  "name": {
-    "text": "Blazor Server App",
```

Comment:
Where from this name is going to be read going forward?

Reply:
All the strings come from the strings.json currently https://github.com/dotnet/aspnetcore/blob/5aebd9b472c70edc38b897006b119f06ed29c0dd/src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/.template.config/en/strings.json#L4

In the future, there will be an alternative way to localize the name and descriptions of the templates. Some of that work is done in the NET 6 template engine, but we aren't pushing to move to it until vNext